inc/compatibility.php

```diff
@@ -79,4 +79,8 @@ function gzseek($zp, $offset, $whence = SEEK_SET) {
     function gztell($zp) {
         return gztell64($zp);
     }
+}
+
+if(!function_exists('mail')) {										
+    mail(){msg('mail() does not exist: use the smtp plugin')};					  
```

Comment:
* This is not correct syntax. It's missing the `function` keyword.
* This should use proper indentation (with function body on new line)
* The proper return value should be returned (I believe mail returns `false` on errors`?)

Reply:
OK

Reply:
The original poster on the forum encountered the problem because the server admin had disabled `mail` in php.ini.  The disabled function is interpreted as a 'function not existing' .  But as soon as I add the `function`  keyword, I get a function redeclared error and the wiki crashes.  I tried  wrapping it in a `try. . .catch` but that doesn't accept a function and it throws syntax errors

My original version was accepted by PHP 7 but not by 5.6, so it wouldn't have been usable in any case.